circleci-orbs/preview.yml

```diff
@@ -0,0 +1,95 @@
+description: Easily register and deploy a component to a new environment on the Architect Cloud, then destroy the component and environment when ready
+
+display:
+  home_url: https://architect.io
+  source_url: https://github.com/architect-team/architect-cli/tree/master/circleci-orbs
+
+jobs:
+  create:
+    parameters:
+      email:
+        type: string
+      password:
+        type: string
+      account:
+        type: string
+      environment:
+        type: string
+      platform:
+        type: string
+      component_name:
+        type: string
+      component_file:
+        type: string
+        default: ./architect.yml
+      deploy_flags:
+        type: string
+        default: ''
+    executor: preview_executor
+    steps:
+      - run: npm install -g @architect-io/cli@0.7
+      - run: architect login -e << parameters.email >> -p << parameters.password >>
+      - run: architect register << parameters.component_file >> -t << parameters.environment >>
+      - run: architect environment:create << parameters.environment >> -a << parameters.account >> --platform << parameters.platform >>
+      - run: architect deploy --auto_approve -a << parameters.account >> -e << parameters.environment >> << parameters.component_name >>:<< parameters.environment >> << parameters.deploy_flags >>
```

Comment:
I'm confused by all of the `>>` `<<` is that really necessary? Any other way to specify run commands?

Reply:
This is the interpolation syntax they use. What would you suggest instead?

Reply:
Gotcha I had no idea it looking at it - their docs don't use spaces, but we do for our own interpolation so I guess this is fine.